features/vote_review.feature

```diff
@@ -68,7 +68,7 @@ Feature: Rate Reviews
     And I enter "5" in the "Professor other thoughts" field
     When I submit my Review form
     When I click the downvote button
-    Then I should see the number of downvotes equal to "1"
+    Then I should see the number of upvotes equal to "0"
```

Comment:
Does a clean review start at 1 `upvote`? Or `0` upvotes? If it starts at `0`, isn't this checking default behavior?